OpenSim/Common/Test/testC3DFileAdapter.cpp

```diff
@@ -21,16 +21,17 @@
  * -------------------------------------------------------------------------- */
 
 #include "OpenSim/Common/C3DFileAdapter.h"
-#include "OpenSim/Common/TRCFileAdapter.h"
 #include "OpenSim/Common/STOFileAdapter.h"
-#include <OpenSim/Auxiliary/auxiliaryTestFunctions.h>
-
-#include <vector>
-#include <unordered_map>
-#include <cstdlib>
+#include "OpenSim/Common/TRCFileAdapter.h"
 #include <chrono>
-#include <thread>
 #include <cmath>
+#include <cstdlib>
+#include <thread>
+#include <unordered_map>
+#include <vector>
+
+#include <OpenSim/Auxiliary/auxiliaryTestFunctions.h>
+#include <OpenSim/Common/Stopwatch.h>
```

Comment:
Any reason for reordering the headers in this PR @chrisdembia ?

Reply:
This reordering is done automatically by ClangFormat.

Reply:
We define the rules for ordering the includes in `.clang-format`.